samples/basic/minimal/mec.conf

```diff
@@ -0,0 +1,41 @@
+# Drivers and peripherals
```

Comment:
Do you need this file? isn't it equivalent to a combination of the arm + no-mt configurations?

Reply:
if i used the common.conf , for mec15xxevb_assy6953 board, it will report a build failure.
for example, if i disabled CONFIG_PINMUX as common.conf,  it failed with 
 "error: 'porta' undeclared (first use in this function)
  152 |  pinmux_pin_set(porta, MCHP_GPIO_000, MCHP_GPIO_CTRL_MUX_F0);"
so i creaed this file for mec board.